utils/bcm27xx-eeprom/patches/0001-rpi-eeprom-update-OpenWrt-defaults.patch

```diff
@@ -0,0 +1,26 @@
+From f3adb004f777088386cc6a830711b46635a7516c Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?=C3=81lvaro=20Fern=C3=A1ndez=20Rojas?= <noltari@gmail.com>
+Date: Mon, 23 Mar 2020 10:10:55 +0100
+Subject: [PATCH] rpi-eeprom-update: OpenWrt defaults
+MIME-Version: 1.0
+Content-Type: text/plain; charset=UTF-8
+Content-Transfer-Encoding: 8bit
+
+Signed-off-by: Álvaro Fernández Rojas <noltari@gmail.com>
+---
+ rpi-eeprom-update-default | 3 ++-
+ 1 file changed, 2 insertions(+), 1 deletion(-)
+
+--- a/rpi-eeprom-update-default
++++ b/rpi-eeprom-update-default
+@@ -1,8 +1,9 @@
+ 
+ FIRMWARE_ROOT=/lib/firmware/raspberrypi/bootloader
+-FIRMWARE_RELEASE_STATUS="critical"
++FIRMWARE_RELEASE_STATUS="stable"
+ FIRMWARE_IMAGE_DIR="${FIRMWARE_ROOT}/${FIRMWARE_RELEASE_STATUS}"
+ FIRMWARE_BACKUP_DIR="/var/lib/raspberrypi/bootloader/backup"
```

Comment:
`/var` is tmpfs - suggest to change this to a more permanent location (and also patch the script's [default value](https://github.com/raspberrypi/rpi-eeprom/blob/a9ca308223c1d0426b9ab320696b95954078c3b4/rpi-eeprom-update#L29)).

Reply:
I don't see the point in patching the script's default value if we're are going to use the default include script.